### DISCUSSION TITLE
Can I get microweber to work on subdomains only?

### DISCUSSION THREAD

**Discussion by abdobargush** [microweber/microweber#958] [Q&A]
Hello,

If I have my laravel app on `mydomain.test` and I want to use microweber's multi site feature but only on subdomains like `second.mydomain.test`

Can this be archived? and how?

**Comment by peter-mw** [comment]
You can host it in wildcard domain and use the multi domain setup  http://docs.microweber.com/guides/installation.md#multi-site-setup
